src/lib/dnssd/platform/Dnssd.h

```diff
@@ -290,6 +291,20 @@ CHIP_ERROR ChipDnssdResolve(DnssdService * browseResult, chip::Inet::InterfaceId
  */
 CHIP_ERROR ChipDnssdResolve(DnssdService * browseResult, chip::Inet::InterfaceId interface,
                             CommissioningResolveDelegate * delegate);
+
+/**
+ * This function resolves the services published by mDNS
```

Comment:
Operational services, right?  And the previous one should talk about commissionable (or commissioner?) services?